public/javascripts/Gallery/css/cards.css

```diff
@@ -149,11 +150,18 @@ with respect to the image holder. This allows the validation menu to be placed o
     width: 100%;
     flex-direction: row;
     flex-wrap: wrap;
+    flex-wrap: nowrap; /* Prevent wrapping */
+    overflow: hidden; /*new*/
 }
 
 .label-tags-content {
-    text-overflow: ellipsis;
+    /* text-overflow: ellipsis;
     width:fit-content;
+    color: black; */
+    text-overflow: ellipsis; /* Add ellipsis to truncated text */
```

Comment:
I also don't think that the comments here are very helpful, I don't think that they really add anything that you don't get from reading the code itself. For example, a comment that says "Ensure the overflow is hidden" doesn't give me more info than reading "overflow: hidden".

The one exception here is for "width: fit-content". I think that your comment explains what that practically means in a way that's helpful!